### ISSUE TITLE
what‘s the meaning of the Sample ratio in the paper

### ISSUE THREAD

**Issue by 1shenhui** [HamPerdredes/SOOD#18]
Dear author, I read your paper, but I have the question about the sample ratio in your paper.
Does this concept have anything to do with the top k% score in the paper dense teacher?

**Comment by HamPerdredes** [comment]
It is similar to the concept in DenseTeacher. Specifically, DenseTeacher selects Top-K from all predictions, while SOOD randomly selects K samples from predictions inside pseudo-boxes (generated by the teacher) area.

**Comment by 1shenhui** [comment]
From the email you replied，you reffered that SOOD randomly selects K samples from predictions inside pseudo-boxes (generated by the teacher) area. Do pseudo-boxes represent the SPL in your paper? I have some question about the predictions inside pseudo-boxes (generated by the teacher) area you said. If the predictions insiade pseudo-boxes area represent post-sigmoid logits, and then you randomly selects K smaples,&nbsp;&nbsp;Will this lead to the selection of a large number of negative samples and low score predictions?




------------------&nbsp;原始邮件&nbsp;------------------
发件人:                                                                                                                        "HamPerdredes/SOOD"                                                                                    ***@***.***&gt;;
发送时间:&nbsp;2023年12月26日(星期二) 上午10:32
***@***.***&gt;;
***@***.******@***.***&gt;;
主题:&nbsp;Re: [HamPerdredes/SOOD] what‘s the meaning of the Sample ratio in the paper (Issue #18)





 
It is similar to the concept in DenseTeacher. Specifically, DenseTeacher selects Top-K from all predictions, while SOOD randomly selects K samples from predictions inside pseudo-boxes (generated by the teacher) area.

**Comment by HamPerdredes** [comment]
This selection strategy does have the possibility of introducing negative samples to loss calculation. However, the motivation behind this operation is keeping the selection process more general. It is kind of like a trade-off choice.
